server/pages/index.js

```diff
@@ -369,8 +371,10 @@ module.exports = function() {
 				const pathRegex = utils.pathToRegExp( path );
 
 				app.get( pathRegex, function( req, res, next ) {
+					req.context = Object.assign( {}, req.context, { sectionName: section.name } );
```

Comment:
@ehg True. I'm thinking, though, that it is helpful to log the response time, categorized by section, whether or not code splitting is enabled.

For example, `/themes` is generally going to take longer to respond than `/`, because `/themes` is server-side rendered. That should be independent of whether the client-side JS is being split or not.

(That said, it occurs to me that `/themes` is only client-side rendered if we're logged out...but either way, I believe it's independent of code splitting. I may be wrong though! 😄  )

What do you think? Does that make sense?